.github/workflows/bitcoinops.yml

```
@@ -25,6 +25,22 @@ jobs:
         run: |
           mkdir /tmp/data
           python bitcoinops/main.py
+
+      - name: Configure Git
+        run: |
+          git config user.email "${{ secrets.GIT_AUTHOR_EMAIL }}"
+          git config user.name "${{ secrets.GIT_AUTHOR_NAME }}"
+
+      - name: Add and commit changes
+        run: |
+          git add .
+          if git diff --staged --quiet; then
+            echo "No changes to commit"
+          else
+            git commit -m "updated scraper logs"
```

Let's change this commit message to be more descriptive by prefixing with the domain/source.
In general we should try to prefix commits with the domain/source that the changes are relevant to.
```suggestion
            git commit -m "bitcoinops: update scraper logs"
```